### DISCUSSION TITLE
Decimal type not compatible to number?

### DISCUSSION THREAD

**Discussion by mpiorowski** [prisma/prisma#5975] [Q&A] [answered]
Why i got a typescript error when trying to assign Postgres Decimal Column into number variable? Is it normal?

**Comment by tarazena** [comment, marked as answer]
Because Prisma started generating [Decimal.js](https://mikemcl.github.io/decimal.js/) objects instead of numbers, you can cast those into numbers by adding `.toNumber()` or `toFixed(2)` to the returning object.

The problem with generating Decimal.js objects though is that if you are returning the objects from Prisma to GraphQL Server, you will see errors because GraphQL server expects Floats and not Decimal Objects. I wonder if there is a way to optionally disable Decimal generation.

**Reply by mpiorowski** [reply]
Great response, thank You :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
